terraform/cloudfoundry/vpc_peering.tf

```diff
@@ -25,3 +25,17 @@ resource "aws_route" "vpc_peer_route_2" {
   destination_cidr_block    = "${element(var.peer_cidrs, count.index)}"
   vpc_peering_connection_id = "${element(aws_vpc_peering_connection.vpc_peer.*.id, count.index)}"
 }
+
+resource "aws_route" "vpc_peer_route_3" {
+  count                     = "${length(var.peer_vpc_ids)}"
+  route_table_id            = "${aws_route_table.internet.3.id}"
+  destination_cidr_block    = "${element(var.peer_cidrs, count.index)}"
+  vpc_peering_connection_id = "${element(aws_vpc_peering_connection.vpc_peer.*.id, count.index)}"
+}
+
+resource "aws_route" "vpc_peer_route_4" {
+  count                     = "${length(var.peer_vpc_ids)}"
+  route_table_id            = "${aws_route_table.internet.4.id}"
+  destination_cidr_block    = "${element(var.peer_cidrs, count.index)}"
+  vpc_peering_connection_id = "${element(aws_vpc_peering_connection.vpc_peer.*.id, count.index)}"
+}
```

Comment:
Why are these 2 blocks added? It should only be necessary to add to the json file to add a new peering connection.

Reply:
I thought we needed a block per VPC added in addition to the VPC that PaaS runs in.

Reply:
This gets a little involved, but we need one route per VPC peering per route-table. Because this 2-dimensional arry-like thing cant be done directly with terraform, we've instead created a single array resource (using the `count` param) per AZ, effectively hard-coding the fact that we have 3 AZs (and therefore 3 route-tables).

The upshot is that this will only need to be changed if we change the number of route-tables we have.

There's more details in the docs here: https://github.com/alphagov/paas-cf/blob/master/doc/vpc_peering.md